tests/integration/test_data/pip_e2e_test/bom.json

```diff
@@ -1,134 +1,134 @@
 {
-  "bomFormat": "CycloneDX",
```

Comment:
Any idea why this file was changed? Seems like all whitespace changes to me

Reply:
Very weird. I kept getting integration test failures, so I regenerated the file, but obviously messed it up...

Reply:
Fixed this, and the one below.